### PR TITLE
Hide books section on home page since books are empty

### LINKED ISSUE #693: [solidjs-tailwind] Hide books section on home page since books are empty
# Background
- Books has a big featured section on the homepage but SolidJS doesn't have any. We should probably consider a change to not make books so heavily featured and switch some homepage items around. Not a blocker but noting here.

Image

### PR DESCRIPTION
**Pull request by vyktoremario**
## Type of change

- [ ] Content addition
- [ ] Bug fix
- [x] Behavior change

## Summary of change

Hide books section on home page since books are empty

- Books has a big featured section on the homepage but SolidJS doesn't have any. We should probably consider a change to not make books so heavily featured and switch some homepage items around. Not a blocker but noting here.

Screenshot 2023-10-18 at 10 47 53

## Checklist

- [ ] This change resolves #693 
- [x] I have confirmed with a maintainer that this change is desired and been
      given the go-ahead to work on it in the relevant issue discussion
- [x] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [x] I have verified the change and the rest of the site works as expected
- [ ] If the change introduces new components, these have been added to
      `packages/system/src/components` with a `.stories.tsx` file that
      adequately renders possible variations of each component.

### CODE REVIEW

File: packages/site/src/data/svelte/blogs.ts [hunk 1]
```diff
@@ -85,16 +85,6 @@ export const blogs: Blog<(typeof blogTags)[number]>[] = [
...
-	{
-		title: 'Rodney Lab',
-		author: 'Rodney',
-		description:
-			'Hello and welcome! I’m Rodney, a web developer based in the UK 🇬🇧 and am available for consultancy work.',
-		image:
-			'https://rodneylab.com/assets/rodney-johnson-about-rodneylab-1350x1350.208d1490.avif',
-		href: 'https://rodneylab.com/tags/sveltekit/',
-		tags: [],
-	},
```
Comment (tvanantwerp) [inline packages/site/src/data/svelte/blogs.ts:97]: Curious, why was this one removed? At first glance, it does look like a bunch of SvelteKit blogs.

File: packages/site/src/data/svelte/blogs.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const blogs: Blog<(typeof blogTags)[number]>[] = [
 		title: 'Joy of Code',
 		author: 'Matija',
 		description:
-			"Matija from 🇭🇷 Croatia and he's infinitely curious at how things work but he's mostly passionate about ☕ JavaScript and 🎨 UI/UX design.",
+			"Matija from Croatia and he's infinitely curious at how things work but he's mostly passionate about JavaScript and UI/UX design.",
```
Comment (tvanantwerp) [inline packages/site/src/data/svelte/blogs.ts:20]: Wondering why emoji are being removed. Rendering issues? I know flags sometimes don't render right depending on the OS and browser—had that problem plenty of times on Windows machines. Not sure if that's still the case, or the reason for removing them in this PR.

File: packages/site/src/data/svelte/communities.ts [hunk 3]
```diff
@@ -201,14 +201,6 @@ export const communities: Community<(typeof communityTags)[number]>[] = [
...
 		tags: ['conferences'],
 	},
-	{
-		name: 'Svelte Summit',
-		description: 'A meetup dedicated to the latest release of Svelte',
-		image: 'https://sveltesociety.dev/images/logo.svg',
-		type: 'Live Events',
-		href: 'https://beta.guild.host/svelte-society-london/events',
-		tags: ['conferences'],
-	},
```
Comment (tvanantwerp) [inline packages/site/src/data/svelte/communities.ts:211]: Looks outdated as a conference, but appears to be a still-active community. Why remove it? Just needs some updating to make it a bit more evergreen.

File: packages/system/src/components/homepage/homepage.tsx [hunk 4]
```diff
@@ -103,9 +103,15 @@ export function Homepage({
...
+				<div
+					className={
+						books.length !== 0
+							? homepageTwoAndOneSectionStyle
+							: ''
+					}
+				>
 					<LatestTools tools={tools} />
-					<Books books={books} />
+					{books.length !== 0 && <Books books={books} />}
```
Comment (tvanantwerp) [inline packages/system/src/components/homepage/homepage.tsx:114]: Could reduce the conditional checks and consolidate like so:

```suggestion
				{books.length > 0 ? (
					<div className="homepageTwoAndOneSectionStyle">
						<LatestTools tools={tools} />
						<Books books={books} />
					</div>
				) : (
					<LatestTools tools={tools} />
				)}
```

**Review by tvanantwerp — CHANGES_REQUESTED** [review]
Would like to know more about why some of the content was removed. It's not part of the linked ticket and I don't see an obvious reason for it.

Also, a suggestion about structuring the conditional inclusion of books.

File: packages/site/src/data/svelte/blogs.ts (hunk 1 above)
Comment (vyktoremario) [reply packages/site/src/data/svelte/blogs.ts:97]: This was a duplicate.

So I had to remove it.

File: packages/site/src/data/svelte/communities.ts (hunk 3 above)
Comment (vyktoremario) [reply packages/site/src/data/svelte/communities.ts:211]: This was a duplicate and the link is not the correct link for Svelte Summit. 
There is another entry that has the correct info.

File: packages/site/src/data/svelte/blogs.ts (hunk 2 above)
Comment (vyktoremario) [reply packages/site/src/data/svelte/blogs.ts:20]: I removed them due to rendering issues. 
They were not rendering correctly on my Mac and I perceive some users might have the same issue. 

I don't think it adds enough value to decide to accommodate it, so I removed it.

**Review by tvanantwerp — APPROVED** [review]

**Pull request merged**